extensions/wikia/PortableInfobox/styles/PortableInfoboxBuilder.scss

```diff
@@ -0,0 +1,37 @@
+@import 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationVariables.scss';
+
+.mw-special-PortableInfoboxBuilder {
+	padding: 0;
+}
+
+.WikiaPage {
+	border: none;
+	.InfoboxBuilder {
+		height: calc(100vh - #{$global-navigation-height});
+		position: fixed;
+	}
+	.WikiaPageBackground {
+		height: 99%;
+	}
+}
+
+@media only screen and (max-width: 1083px) {
+	.WikiaPage, .InfoboxBuilder {
+		padding: 0;
+		width: 766px;
```

Comment:
Can existing width variables and mixing related to breakpoints be used to calculate the width values?
( see breakpoints-variables.scss )